data/campaigns/Eastern_Invasion/units/Elder_Jinn.cfg

```diff
@@ -0,0 +1,197 @@
+#textdomain wesnoth-ei
+
+# TODO: Elder Jinn needs its own sprite
+#define IMG_PATH_TEMP
+units/monsters/jinn#enddef
+
+[unit_type]
+    id=Elder Jinn
+    name= _ "Elder Jinn"
+    race=monster
+    {TRAIT_ELEMENTAL}
+    image="{IMG_PATH_TEMP}/jinn.png"
+    profile="portraits/monsters/jinn.webp"
+    hitpoints=75
+    movement_type=spirit
+    movement=3
+    experience=100
+    level=3
+    alignment=neutral
+    advances_to=null
+    {AMLA_DEFAULT}
+    [abilities]
+        {ABILITY_SELF_HEAL}
+    [/abilities]
+    cost=90
+    usage=archer
+    #po: currently the same as regular Jinn's description
```

Comment:
Instead of a po comments, shouldn't this just be a textdomain declaration? Either that or rewrite the description before merging (and remove the comment).